prediction_market_agent_tooling/markets/seer/data_models.py

```
@@ -186,3 +187,13 @@ def created_time(self) -> DatetimeUTC:
         chain_id = RPCConfig().chain_id
         return urljoin(SEER_BASE_URL, f"markets/{chain_id}/{self.id.hex()}")
+
+
+class RedeemParams(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+    market: ChecksumAddress
+    outcome_indices: list[int] = Field(alias="outcomeIndexes")
+    amounts: list[OutcomeWei]
+
+
+ADDRESS_ZERO
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)